src/config/src/meta/stream.rs

```diff
@@ -878,6 +879,10 @@ impl RoutingCondition {
                     _ => false,
                 }
             }
+            Value::Null => {
+                matches!(self.operator, Operator::EqualTo)
+                    && matches!(&self.value, Value::String(v) if v == "null")
+            }
```

Comment:
In case of `Value::Null`, this ensures only when condition is `=` and user explicitly enters `null`, can this evaluate to true and continue. in all other cases, this evaluate to false.